pkg/3scale/amp/operator/backend_reconciler.go

```diff
@@ -5,6 +5,9 @@ import (
 	"github.com/3scale/3scale-operator/pkg/3scale/amp/component"
 	"github.com/3scale/3scale-operator/pkg/helper"
 	"github.com/3scale/3scale-operator/pkg/reconcilers"
+	v1 "k8s.io/api/core/v1"
```

Comment:
same as above comment

Reply:
sure can do

Reply:
Goland disagrees with you on this one, keeps reverting it :facepalm: